### DISCUSSION TITLE
Question on GPP output

### DISCUSSION THREAD

**Discussion by ATran001** [martinpaljak/GlobalPlatformPro#346] [Q&A]
Hi I'm pretty new to using GPP and programming simcards in general.

I used this command: gp list and got this response:

[WARN] GPSession - Could not parse SELECT response: Index 7 out of bounds for length 7
Error: Could not auto-detect ISD AID: A0000000030000

Is this the expected response? I'm thinking this happened because there is no applets on the card?

Any help or advice is appreciated. Thanks!

**Comment by martinpaljak** [comment]
Please provide a full log (with -d). Please note that many (most?) SIM cards are not global platform cards in the context of GP card spec, meaning that GPPro can not be used to manage applets on those.

**Reply by ATran001** [reply]
@martinpaljak Hi Martin, thanks for responding to my question! Hopefully my hardware isn't incompatible! Could it be my installation of GPPro, I'm using the newest version of Java?  

I ran the command with the -d argument and this is what I got:

gp -list -d
GlobalPlatformPro v21.12.31-55-g52c3581
Running on Windows 11 10.0 amd64, Java 21.0.2 by Oracle Corporation
SCardConnect("Alcorlink USB Smart Card Reader 0", T=*) -> T=0, 3B9E96801FC78031E073FE211B66D001A0DD12006E
A>> T=0 (4+0000) 00A40400 00
A<< (0007+2) (69ms) A0000000030000 9000
[WARN] GPSession - Could not parse SELECT response: Index 7 out of bounds for length 7
Error: Could not auto-detect ISD AID: A0000000030000
SCardDisconnect("Alcorlink USB Smart Card Reader 0", false) tx:5/rx:9 in 93ms

**Reply by ATran001** [reply]
For further context on the hardware I'm testing on, I'm using a Gemalto SIM card

**Reply by ATran001** [reply]
@martinpaljak Hi Martin wanted to follow up on the supported cards. In the document here: https://github.com/martinpaljak/GlobalPlatformPro/tree/master/docs/TestedCards, it states that Gemalto Cards should be supported. In this case what do you think the issue could be in reading this card. Are there any other commands prior to running gp -list? Thanks!
